elm/src/test/resources/application-fuzzy-test-1.conf

```diff
@@ -5,7 +5,7 @@ elm {
   }
 
   forging {
-    delay = 2s
+    delay = 5s
```

Comment:
We could have 1 config file named `common.conf` which would be a fallback for the node-specific ones. Something like:

```
val commonConf = ConfigFactory.load("common.conf")
AppConfig.load(ConfigFactory.load("application-fuzzy-test-1.conf").withFallback(commonConf))
```

Could be done later, though.